jenkins/debian/Jenkinsfile_igt

```diff
@@ -0,0 +1,43 @@
+library identifier: 'jenkins-libs@master', retriever: modernSCM(
+  [$class: 'GitSCMSource',
+   remote: 'https://github.com/ana/kernelci-jenkins-libs'])
+
+buildTest {
+
+    name = 'igt'
+    archList = ["arm64", "armel", "x86_64"] //"x86" doesn't work because libdrm FTBFS
+    debosFile = "jenkins/debian/debos/stretch.yaml"
+    //These packages will be installed in the base image, they're the runtime deps of the test suite
+    extra_packages = "libpciaccess0 libkmod2 libprocps6 libcairo2 libunwind8 libudev1 libglib2.0-0"
+
+    // This script must build the test suite.
+    // The available environment variables are: ${PIPELINE_VERSION}, ${ARCH}, ${TOOLCHAIN_ARCH}, ${NAME}
+    // The script must produce a tarball with the built test suite named rootfs-built-${NAME}-${ARCH}.cpio.gz
+    build_test_suite = '''
+# Build DRM
+mkdir -p ${WORKSPACE}/${PIPELINE_VERSION}/${ARCH}/libdrm
+cd ${WORKSPACE}/${PIPELINE_VERSION}/${ARCH}/libdrm/
+git clone git://anongit.freedesktop.org/mesa/drm .
+mkdir -p usr # Build to usr so it's in the right place when we add it to cpio
+PKG_CONFIG_PATH=/usr/lib/${TOOLCHAIN_ARCH}/pkgconfig/ sh autogen.sh --host=${TOOLCHAIN_ARCH} --enable-intel --prefix=\$(pwd)/usr/
+make -j\$(nproc) V=1
+make -j\$(nproc) install V=1
+ls usr/lib/lib*.so* | cpio -H newc -v -o  > ../rootfs-built-$NAME-${ARCH}.cpio
+
+
+# Build Intel GPU tools
```

Comment:
We should avoid mentioning Intel in this context, as the test suite isn't Intel-specific any more and has been renamed to "IGT GPU Tools"